src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-put.ts

```diff
@@ -1,17 +1,27 @@
-import { NextFunction, Request, Response } from "express";
+import { Request, Response, NextFunction } from "express";
+import { GitHubServerApp } from "~/src/models/github-server-app";
 
 export const JiraConnectEnterpriseAppPut = async (
 	req: Request,
 	res: Response,
 	next: NextFunction
 ): Promise<void> => {
+	req.log.debug("Received Jira Connect Enterprise App PUT request to update app.");
```

Comment:
Moved this out of the try/catch. If the request fails, we don't see this log and can't be certain we hit this middleware (think this needs to be updated in other places as well).